sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -258,8 +255,8 @@ public ServiceBusReceivedMessage peekMessage() {
 
     /**
      * Reads the next active message without changing the state of the receiver or the message source. The first call to
-     * {@code peek()} fetches the first active message for this receiver. Each subsequent call fetches the subsequent
-     * message in the entity.
+     * {@code peekMessage()} fetches the first active message for this receiver. Each subsequent call fetches the
+     * subsequent message in the entity.
```

Comment:
Document : This should throw exception if this is called for non-session entity. I think we do not have that check. we should add that check.

Reply:
This is done on the management node. You don't have to be a sessioned receiver to receive a message from it.